source/qdk_package/README.md

```diff
@@ -0,0 +1,153 @@
+# qdk
+
+Experimental meta-package for the Quantum Development Kit (QDK) that bundles the existing
+`qsharp` Python package together with optional extras under a single, stable import root: `import qdk`.
+
+The design is intentionally minimal: submodules plus import-time detection of optional components.
+
+## Install
+
+Base (always includes `qsharp`):
+
+```bash
+pip install qdk
+```
+
+Jupyter extra (bundles widgets + JupyterLab extension package — provides only the `qdk.widgets` Python surface):
+
+```bash
+pip install qdk[jupyter]
+```
+
+Azure Quantum extra (adds `azure-quantum`):
+
+```bash
+pip install qdk[azure]
+```
+
+Qiskit extra (adds `qiskit`):
+
+```bash
+pip install qdk[qiskit]
+```
+
+All extras:
+
+```bash
+pip install qdk[all]
+```
+
+## Quick Start
+
+```python
+from qdk import qsharp
+
+result = qsharp.run("{ use q = Qubit(); H(q); return MResetZ(q); }", shots=100)
+print(result)
+```
+
+Widgets (installed via jupyter extra):
+
+```python
+try:
+    from qdk import widgets  # requires: pip install qdk[jupyter]
+    # Use widgets per qsharp-widgets documentation
+except ImportError:
+    widgets = None  # Optional: feature not installed
+```
+
+Qiskit (if installed):
+
+```python
+try:
+    from qdk import qiskit  # requires: pip install qdk[qiskit]
+    qk = qiskit
+    # Example: qk.transpile(...)
+except ImportError:
+    qk = None
```

Comment:
I think this readme needs a lot of work, and I don't think the patterns shown here make sense (i.e. wrapping imports in exception handling and reassigning the module to a local). But lets get this in and clean it up after.